debian/copyright

```diff
@@ -33,7 +33,7 @@ following license:
  */
 
 On Debian GNU/Linux systems, the text of the LGPL can be found under
-/usr/share/common-licenses/LGPL-2.1
+'/usr/share/common-licenses/LGPL-2.1'
```

Comment:
not sure this helps much (this file is not using the machine-parseable format of pain), but I suppose it doesn't do harm.

Reply:
Well it's generating https://lintian.debian.org/tags/copyright-refers-to-symlink-license.html as we're supposedly pointing to usr/share/common-licenses/GPL. That's the only reference I could see to GPL$. Actually I guess you're referring to the random addition of extra ' by me. I did wonder, but given we've got them on the last line...

Is it worth us trying to move to the machine-readable format at some point?

Reply:
the GPL$ is indeed something else, which you (correctly) fixed. The addition of ' seems somewhat superfluous, but it doesn't do any harm.

I'm not convinced of the benefit of the machine-parseable copyright files. They require a *lot* of work for little gain. To me, copyright files are a necessary evil for uploading anything into Debian, I'd rather not spend too much time on them ;-)

Reply:
Ah okay, I think I just added them because the last one has them and I like consistency. I'm happy to remove the ' from all of the mentions of copyright files if you think that's more appropriate?

Okay, I'm happy to skip the machine option until Lintian starts moaning then!